code/obj/item/clothing/gloves.dm

```diff
@@ -340,8 +340,8 @@ ABSTRACT_TYPE(/obj/item/clothing/gloves)
 			return
 
 /obj/item/clothing/gloves/swat
-	desc = "A pair of syndicate tactical gloves that are quite fire and electrically-resistant. They also help you block attacks. They do not specifically help you block against blocking though. Just regular attacks."
-	name = "SWAT Gloves"
+	desc = "A pair of Syndicate tactical gloves that are electrically insulated and quite heat-resistant. The high-quality materials help prevent ne'er-do-wells from knocking stuff out of your hands."
```

Comment:
i think i follow dim here on this one; "help block attacks" is a little more mechanically straight-forward since it's a blocking system, even if it is a bit unimmersive as a description to go "hey this thing can be used in this mechanical system". agree with the cutting off of the last two sentences though, they don't really do anything imo other than get confusing